src/server/routes/api/thumbnail.js

```diff
@@ -0,0 +1,155 @@
+import Express from 'express';
+import Cytoscape from 'cytoscape';
+import VizMapper from '../../../model/vizmapper';
+import { DEFAULT_NODE_STYLE, DEFAULT_EDGE_STYLE } from '../../../model/style';
+import fetch from 'node-fetch';
+import { getNetworkDocURL, getSnapshotDocURL, removeQuotes } from './history';
+
+// TODO Rewrite this to use 'import'
+const cytosnap = require('cytosnap');
```

Comment:
`import cytosnap from 'cytosnap'` should work

Reply:
Ok will do.